routers/repo/issue.go

```diff
@@ -1542,6 +1548,11 @@ func ChangeCommentReaction(ctx *context.Context, form auth.ReactionForm) {
 
 	switch ctx.Params(":action") {
 	case "react":
+		if !util.IsStringInSlice(form.Content, setting.UI.Reactions) {
+			log.Error("ChangeIssueReaction: Reaction '%s' is no allowed reaction", form.Content)
```

Comment:
Same as the other comment.

(And same small wording nit)
```suggestion
			log.Error("ChangeIssueReaction: '%s' is not an allowed reaction", form.Content)
```

Reply:
done :+1: